### PR TITLE
feat: set up repo for samples

### PR DESCRIPTION
**Pull request by anguillanneuf**
Modified these files to set up samples testing. A follow-up PR (#197) will add samples & tests.

```
        new file:   .kokoro/continuous/java11-samples.cfg
        new file:   .kokoro/continuous/jave8-samples.cfg
        new file:   .kokoro/nightly/java11-samples.cfg
        new file:   .kokoro/nightly/jave8-samples.cfg
        new file:   .kokoro/presubmit/java11-samples.cfg
        new file:   .kokoro/presubmit/jave8-samples.cfg
        new file:   .kokoro/run_samples_tests.sh
        modified:   pom.xml
```

### CODE REVIEW

File: .kokoro/presubmit/jave8-samples.cfg [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    key: "TRAMPOLINE_IMAGE"
+    value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+}
+
+# Tell trampoline which tests to run.
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/java-pubsub/.kokoro/run_samples_tests.sh"
+}
+© 2020 GitHub, Inc.
```
Comment (chingor13) [inline .kokoro/presubmit/jave8-samples.cfg:31]: This looks like a typo

File: .kokoro/nightly/jave8-samples.cfg [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    key: "TRAMPOLINE_IMAGE"
+    value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+}
+
+# Tell trampoline which tests to run.
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/java-pubsub/.kokoro/run_samples_tests.sh"
+}
+© 2020 GitHub, Inc.
```
Comment (chingor13) [inline .kokoro/nightly/jave8-samples.cfg:31]: Typo?

File: .kokoro/continuous/jave8-samples.cfg [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    key: "TRAMPOLINE_IMAGE"
+    value: "gcr.io/cloud-devrel-kokoro-resources/java8"
+}
+
+# Tell trampoline which tests to run.
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/java-pubsub/.kokoro/run_samples_tests.sh"
+}
+© 2020 GitHub, Inc.
```
Comment (chingor13) [inline .kokoro/continuous/jave8-samples.cfg:31]: Typo?

File: .kokoro/presubmit/jave8-samples.cfg (hunk 1 above)
Comment (anguillanneuf) [reply .kokoro/presubmit/jave8-samples.cfg:31]: Oh no, I need to take it out.

Comment (anguillanneuf) [reply .kokoro/presubmit/jave8-samples.cfg:31]: Please review #201

**Pull request merged**